packages/lesswrong/components/shortform/ShortformThreadList.tsx

```diff
@@ -11,7 +11,7 @@ const styles = (theme: ThemeType): JssStyles => ({
 const ShortformThreadList = ({ classes }: {
   classes: ClassesType,
 }) => {
-  const { LoadMore, CommentWithReplies, ShortformSubmitForm } = Components
+  const { LoadMore, CommentOnPostWithReplies, ShortformSubmitForm } = Components
```

Comment:
I have tested that all of these `CommentOnPostWithReplies` still work as expected with the refactoring

Reply:
Nice work